src/server.c

```diff
@@ -788,8 +788,8 @@ int clientsCronResizeQueryBuffer(client *c) {
 
 /* If the client has been idle for too long, free the client's arguments. */
 int clientsCronFreeArgvIfIdle(client *c) {
-    /* If the arguments have already been freed or are still in use, exit ASAP. */
-    if (!c->argv || c->argc) return 0;
+    /* If the client is in the mid of parsing command, exit ASAP. */
```

Comment:
```suggestion
    /* If the client is in the mid of parsing and processing command, exit ASAP. */
```
after parsing, commands may not always be processed immediately, for example, the commands are from io threads.

Reply:
Update with [`b6105b9` (#13725)](https://github.com/redis/redis/pull/13725/commits/b6105b9c86cf67b94591036c7f2b8d333b54792c)